configs/GEPRC_TAKER_H743/config.h

```diff
@@ -0,0 +1,146 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        GEPRC_TAKER_H743
+#define MANUFACTURER_ID   GEPR
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_BARO_MS5611
+// #ifndef USE_MAG
+#define USE_MAG
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
+// #endif
+#define USE_SDCARD
+#define USE_MAX7456
+
+#define BEEPER_PIN           PD2
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PB5
+#define MOTOR4_PIN           PB4
+#define MOTOR5_PIN           PD12
+#define MOTOR6_PIN           PD13
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
+#define LED_STRIP_PIN        PA8
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART4_TX_PIN         PA0
+#define UART6_TX_PIN         PC6
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PA1
+#define UART6_RX_PIN         PC7
+#define UART7_RX_PIN         PE7
+#define UART8_RX_PIN         PE0
+#define I2C1_SCL_PIN         PB8
+#define I2C2_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB9
+#define I2C2_SDA_PIN         PB11
+#define LED0_PIN             PC13
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PC10
+#define SPI4_SCK_PIN         PE2
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PC11
+#define SPI4_SDI_PIN         PE5
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PC12
+#define SPI4_SDO_PIN         PE6
+#define ADC_VBAT_PIN         PC3
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC2
+#define ADC_EXTERNAL1_PIN    PC1
+#define SDCARD_SPI_CS_PIN    PA15
+#define PINIO1_PIN           PE13
+#define PINIO2_PIN           PC14
+#define MAX7456_SPI_CS_PIN   PE4
+#define GYRO_1_EXTI_PIN      PD0
+#define GYRO_2_EXTI_PIN      PD8
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PB12
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PA8 , 1, 14) \
+    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 2, PB1 , 2,  1) \
+    TIMER_PIN_MAP( 3, PB4 , 1,  4) \
+    TIMER_PIN_MAP( 4, PB5 , 1,  5) \
+    TIMER_PIN_MAP( 5, PD12, 1,  6) \
+    TIMER_PIN_MAP( 6, PD13, 1,  7) \
+    TIMER_PIN_MAP( 7, PC8 , 2,  2) \
+    TIMER_PIN_MAP( 8, PC9 , 2,  3)
```

Comment:
```suggestion
    TIMER_PIN_MAP( 0, PA8 , 1,  0) \
    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
    TIMER_PIN_MAP( 3, PB4 , 1,  0) \
    TIMER_PIN_MAP( 4, PB5 , 1,  0) \
    TIMER_PIN_MAP( 5, PD12, 1,  0) \
    TIMER_PIN_MAP( 6, PD13, 1,  0) \
    TIMER_PIN_MAP( 7, PC8 , 2,  0) \
    TIMER_PIN_MAP( 8, PC9 , 2,  0)
```

Reply:
I need to run some tests first.

Reply:
![image](https://github.com/user-attachments/assets/b8800be5-2f5c-4f42-8d4b-3dcd47d4e12a)
According to your modifications, after recompiling the firmware, it cannot get the 8 motors to work properly.